var/spack/repos/builtin/packages/hbm-dramsim2/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class HbmDramsim2(MakefilePackage):
+    """
+    HBM Simulator based on DRAMSim2
+    """
+
+    homepage = "https://github.com/tactcomplabs/HBM"
+    git = "https://github.com/tactcomplabs/HBM"
+    url = "https://github.com/tactcomplabs/HBM/archive/hbm-1.0.0-release.tar.gz"
+
+    version('1.0.0', sha256="0efad11c58197edb47ad1359f8f93fb45d882c6bebcf9f2143e0df7a719689a0")
+
+    def build(self, spec, prefix):
+        make()
```

Comment:
No need to override `build` here, this is the default behavior.

Reply:
fixed in all packages